HomeWork1/main.swift

```diff
@@ -7,5 +7,135 @@
 
 import Foundation
 
-print("Hello, World!")
+//Создать цикл в цикле. В первом цикле интервал 200 раз во втором как доедем до 15 выйти со всех циклов:)
 
+
+calculation: for _ in 0...200 {
+    for i in 0..<20 {
```

Comment:
```suggestion
    for index in 0..<20 {
```
чтож мы никак не победим) Всегда и везде надо давать полный нейминг. Никаких пока сокращений)